README.md

```diff
@@ -61,6 +61,15 @@ the path to the key file, for example:
 If you are deploying to App Engine, Compute Engine, or Container Engine, your
 credentials will be available by default.
 
+Some of the configurations expect the `GCP_CREDENTIALS` system variable.
```

Comment:
@zeroasterisk What configurations? If I'm a user, how do I know whether I need to set this?

Reply:
good question - 2 of the client's have a test suite which includes this... perhaps the better answer is to normalize all the test suites to the same consistent configuration.

Reply:
```
$ ag GCP_CREDENTIALS
clients/translate/config/config.exs
31:config :goth, json: {:system, "GCP_CREDENTIALS"}

clients/storage/config/config.exs
31:config :goth, json: {:system, "GCP_CREDENTIALS"}
```

Perhaps we should ask @chingor13 why these are configured as a JSON string ENV variable... is there a reason? I'm tempted to recommend we either only use the .json file or only use the ENV... having both may be confusing, unless one explicitly overrides the other and we can say why.

Reply:
This is how goth configuration works (via the application env) - it expects the config to be the json contents vs. the path to the json file. We could send a PR to add a `file:` option that reads the json file.

Reply:
Also, IIUC, these configurations are only used within this repo - when the library is published, this has no bearing on the user's application.